src/panel/direction/DirectionPanel.jsx

```diff
@@ -353,6 +354,7 @@ export default class DirectionPanel extends React.Component {
               fitContent={['default']}
               marginTop={this.marginTop}
               minimizedTitle={_('Unfold to show the results', 'direction')}
+              onClose={this.onClose}
```

Comment:
We should check with the design but I think we shouldn't use the same `onClose`. Here what we want is probably to clear the current result but go back to the form.